interfaces/HO-Portal/src/app/program/phase-next/phase-next.component.html

```diff
@@ -1,5 +1,13 @@
-  <ion-button *ngIf="btnAvailable"
-              [disabled]="selectedPhase !== activePhase"
-              (click)="advancePhase(activePhaseId)"
-              [innerHTML]="btnText"
-  ></ion-button>
+<confirm-prompt *ngIf="btnAvailable"
+                [disabled]="selectedPhase !== activePhase || !phaseReady || isInProgress || currentUserRole !== userRoleEnum.ProgramManager"
```

Comment:
I would put all this logic into a specific method/property IN the `phase-next.component.ts`;
That way not everything needs to be 'made public' to the template as well. (things like the Role, the userRoleEnum, etc.)